.gitlab-ci.yml

```diff
@@ -0,0 +1,64 @@
+stages:
+  - test
+  - build
+
+image:                             parity/rust:nightly
```

Comment:
Not sure the implication of `nightly` tag here. However, for `rust`, stable version should still be the default -- we only use `nightly` to build the wasm runtime, and build everything else on `stable`.

Reply:
This image should have both actually. The reason to keep both inside is that we avoid cross-link errors when attempting to do a rustup on an existing docker image.